lib/timeline/Core.js

```diff
@@ -392,13 +392,7 @@ class Core {
    */
   setOptions(options) {
     if (options) {
-      // copy the known options
-      const fields = [
-        'width', 'height', 'minHeight', 'maxHeight', 'autoResize',
-        'start', 'end', 'clickToUse', 'dataAttributes', 'hiddenDates',
-        'locale', 'locales', 'moment', 'preferZoom', 'rtl', 'zoomKey', 'horizontalScroll', 'verticalScroll'
-      ];
-      util.selectiveExtend(fields, this.options, options); 
+      this.options = { ...this.options, ...options }
```

Comment:
I think we should keep the `selectiveExtend` it ensures that no other options get overwritten (e.g. by mistake).